doc/Publishing.md

```diff
@@ -87,6 +87,24 @@ For example, if you picked `0.1.0` as a version then you should run:
 
 To confirm that the tagging was correctly done, check the repo's [releases](https://github.com/eclipse-theia/theia/releases) page and confirm the release just done is listed there.
 
+## Updating the release page on eclipse
+- Login to eclipse: https://projects.eclipse.org/projects/ecd.theia
+- On the right side panel, select Release / Create a new release
+    Create a new version ex: 1.2.0
+    Select the edit tab
+    - In "The Basic" section
+        - Select the  "Type A" in the IP Due Diligence type
+    - In the "Project Plan"
+        - Deliverables section
+            - Copy from the changelog.md the section about the updates
+        - Compatibility section
+            - Copy the Braking changes from the changelog.md
+- When completing the edition, select "Save" at the bottom of the page.
+- To confirm the data is available,
+    - Open the page https://projects.eclipse.org/projects/ecd.theia
+        - Select the version you just created
+        - Open the Review plan section, you should see the data provided before.
```

Comment:
There are times in the pull-request where punctuation (ex: period **.**) is included and other times when it is not, do you mind updating the pull-request so we are consistent (either we include it or do not).

Reply:
Maybe include a mention that this part should be done in advance of the release? Release reviews are no longer required for every release, just once a year, but when we need to do it, it will need to be scheduled at least 1 week in advance, to give time to the Foundation to review and approve. Next time I think is around August/September.

Reply:
I agree with you @marcdumais-work this should be done 1 week prior to the release, but in the current situation, we accept changes to the release up to the last minutes. If we freeze the content one week before, I guess it would mean we acept new code up to one week before the release .

Reply:
I think it would be ok to a pass post-release. One week before you paste what's there, and update after the release